diff --git a/node_modules/react-github-calendar/dist/index.es.js b/node_modules/react-github-calendar/dist/index.es.js
index 6d517ec..fdc58cc 100644
--- a/node_modules/react-github-calendar/dist/index.es.js
+++ b/node_modules/react-github-calendar/dist/index.es.js
@@ -276,7 +276,7 @@ const GitHubCalendar = ({
 
     return monthLabels.map(month => /*#__PURE__*/React.createElement("text", {
       x: (blockSize + blockMargin) * month.x,
-      y: fontSize,
+      y: (blockSize + blockMargin) * 8,
       key: month.x,
       style: style
     }, month.label));
@@ -287,7 +287,7 @@ const GitHubCalendar = ({
     const textHeight = Math.round(fontSize * LINE_HEIGHT);
     return blocks.map(week => week.map((day, y) => /*#__PURE__*/React.createElement("rect", {
       x: "0",
-      y: textHeight + (blockSize + blockMargin) * y,
+      y: (blockSize + blockMargin) * y,
       width: blockSize,
       height: blockSize,
       fill: theme[`grade${day.info ? day.info.level : 0}`],
@@ -304,9 +304,14 @@ const GitHubCalendar = ({
     return /*#__PURE__*/React.createElement("div", {
       className: getClassName('meta'),
       style: {
-        fontSize
+        fontSize: 17,
+        fontWeight: 'bold',
+        display: 'flex',
+        justifyContent: 'center',
+        textAlign: 'center',
+        paddingBottom: 13
       }
-    }, isCurrentYear && fullYear ? 'Last year' : year, ' – ', isCurrentYear && !fullYear ? 'So far ' : null, totalCount, " contributions");
+    }, totalCount, " Github contributions in the last year");
   }
 
   const {
@@ -338,7 +343,7 @@ const GitHubCalendar = ({
     return /*#__PURE__*/React.createElement("div", {
       key: year,
       className: getClassName('chart', styles.chart)
-    }, /*#__PURE__*/React.createElement("svg", {
+    }, showTotalCount && renderTotalCount(year, totalCount), /*#__PURE__*/React.createElement("svg", {
       xmlns: "http://www.w3.org/2000/svg",
       width: width,
       height: height,
@@ -347,7 +352,7 @@ const GitHubCalendar = ({
       style: {
         backgroundColor: theme === null || theme === void 0 ? void 0 : theme.background
       }
-    }, renderMonthLabels(monthLabels), renderBlocks(blocks)), showTotalCount && renderTotalCount(year, totalCount), children);
+    }, renderMonthLabels(monthLabels), renderBlocks(blocks)), children);
   }));
 };
 
diff --git a/node_modules/react-github-calendar/dist/index.js b/node_modules/react-github-calendar/dist/index.js
index 43ca3d8..fbcca53 100644
--- a/node_modules/react-github-calendar/dist/index.js
+++ b/node_modules/react-github-calendar/dist/index.js
@@ -283,7 +283,7 @@ const GitHubCalendar = ({
 
     return monthLabels.map(month => /*#__PURE__*/React__default.createElement("text", {
       x: (blockSize + blockMargin) * month.x,
-      y: fontSize,
+      y: (blockSize + blockMargin) * 8,
       key: month.x,
       style: style
     }, month.label));
@@ -294,7 +294,7 @@ const GitHubCalendar = ({
     const textHeight = Math.round(fontSize * LINE_HEIGHT);
     return blocks.map(week => week.map((day, y) => /*#__PURE__*/React__default.createElement("rect", {
       x: "0",
-      y: textHeight + (blockSize + blockMargin) * y,
+      y: (blockSize + blockMargin) * y,
       width: blockSize,
       height: blockSize,
       fill: theme[`grade${day.info ? day.info.level : 0}`],
@@ -311,9 +311,14 @@ const GitHubCalendar = ({
     return /*#__PURE__*/React__default.createElement("div", {
       className: getClassName('meta'),
       style: {
-        fontSize
+        fontSize: 17,
+        fontWeight: 'bold',
+        display: 'flex',
+        justifyContent: 'center',
+        textAlign: 'center',
+        paddingBottom: 13
       }
-    }, isCurrentYear && fullYear ? 'Last year' : year, ' – ', isCurrentYear && !fullYear ? 'So far ' : null, totalCount, " contributions");
+    }, totalCount, " Github contributions in the last year");
   }
 
   const {
@@ -345,7 +350,7 @@ const GitHubCalendar = ({
     return /*#__PURE__*/React__default.createElement("div", {
       key: year,
       className: getClassName('chart', styles.chart)
-    }, /*#__PURE__*/React__default.createElement("svg", {
+    }, showTotalCount && renderTotalCount(year, totalCount), /*#__PURE__*/React__default.createElement("svg", {
       xmlns: "http://www.w3.org/2000/svg",
       width: width,
       height: height,
@@ -354,7 +359,7 @@ const GitHubCalendar = ({
       style: {
         backgroundColor: theme === null || theme === void 0 ? void 0 : theme.background
       }
-    }, renderMonthLabels(monthLabels), renderBlocks(blocks)), showTotalCount && renderTotalCount(year, totalCount), children);
+    }, renderMonthLabels(monthLabels), renderBlocks(blocks)), children);
   }));
 };
 
